app/controllers/groups_controller.rb

```diff
@@ -1,5 +1,8 @@
 class GroupsController < ApplicationController
+  require './app/services/create_membership'
+  require './app/services/create_post'
```

Comment:
Please don't use `require`, it's usually unnecessary in rails, because it uses autoloading. If you are interested in, here is some further explanation on autoloading and its behavior
https://guides.rubyonrails.org/v5.0/autoloading_and_reloading_constants.html

Reply:
You are right, I removed it. Thanks for the article, definitely going to read it!